hikari/impl/event_factory.py

```diff
@@ -328,12 +329,19 @@ def deserialize_presence_update_event(
             if "public_flags" in user_payload:
                 flags = user_models.UserFlag(user_payload["public_flags"])
 
+            if ac := user_payload.get("accent_color", None):
+                accent_color: undefined.UndefinedOr[colors.Color] = colors.Color(ac)
+            else:
+                accent_color = undefined.UNDEFINED
```

Comment:
What if the accent_color is None, why are we defaulting it to `undefined.UNDEFINED`?

Also:
```suggestion
            accent_color: undefined.UndefinedOr[colors.Color]
            if ac := user_payload.get("accent_color", None):
                accent_color = colors.Color(ac)
            else:
                accent_color = undefined.UNDEFINED
```

Reply:
Oops. My bad :)

Reply:
Should I use `undefined.UNDEFINED` or `None` if `accent_color` is unavailable?

Reply:
You might have to investigate under what cases `accent_color` is not sent. If they dont send it when its not set, then default it to `None`. If there are cases where they dont send it even if its set, then `undefined.UNDEFINED`.

Reply:
Seems like discord sends banner and accent colour only when fetched from the endpoint. It shall be `undefined.UNDEFINED` then.